src/modules/pass.js

```diff
@@ -227,11 +227,12 @@ PassFF.Pass = (function () {
         }
         return browser.runtime.sendNativeMessage("passff", args)
           .then((result) => {
-            let version = result.version || "0.0";
-            if (version !== "1.0.1" && version !== "testing") {
+            let version = String(result.version) || "0.0";
+            const COMPATIBLE_VERSION = "1.0.1";
+            if (version < COMPATIBLE_VERSION && version !== "testing") {
```

Comment:
I don't think that this check does what you want. Just look at `"1.9" < "1.10"`which returns `false`